website/docs/r/servicebus_subscription.html.markdown

```diff
@@ -69,7 +69,7 @@ The following arguments are supported:
     [TimeSpan](#timespan-format) format.
 
 * `default_message_ttl` - (Optional) The TTL of messages sent to this Subscription
-    if no TTL value is set on the message itself. Provided in the [TimeSpan](#timespan-format)
+    if no TTL value is set on the message itself. Provided in the ISO8601 [TimeSpan](#timespan-format)
```

Comment:
It is not necessary to provide a link for ISO8601 since it is well-knowned format. So we can replace the wording a bit:
```suggestion
    in ISO8601 format, if no TTL value is set on the message itself.
```